router/core/operation_processor.go

```diff
@@ -37,11 +37,9 @@ import (
 	"github.com/wundergraph/cosmo/router/pkg/config"
 )
 
-var (
-	// staticOperationName is used to replace the operation name in the document when generating the operation ID
-	// this ensures that the operation ID is the same for the same operation regardless of the operation name
-	staticOperationName = []byte("O")
-)
+// staticOperationName is used to replace the operation name in the document when generating the operation ID
+// this ensures that the operation ID is the same for the same operation regardless of the operation name
+var staticOperationName = []byte("O")
```

Comment:
please do not do unnecessary changes
it is hard to review and blame will always show you after that ;)

Reply:
I can't save the file without this happening, I don't know how or why this file got into `main` unformatted.

Reply:
go fmt is not modifying this, it is something else

Reply:
[`gofumpt`](https://github.com/mvdan/gofumpt) is doing it, I can disable it for now.

Reply:
Even switching VSCode from `gofumpt` to `gofmt` it still does this, not sure if I can prevent it.

Reply:
Fixed